modules/assisted-installer-adding-hosts-to-the-cluster.adoc

```diff
@@ -0,0 +1,25 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="adding-hosts-to-the-cluster_{context}"]
+= Adding hosts to the cluster
+
+After setting the cluster details and optionally configuring static networking, you must add one or more hosts to the cluster. Adding a host to the cluster involves generating a discovery ISO. The discovery ISO contains an installation of RHCOS and installs an agent on the host. For each host, use the following procedure.
+
+.Procedure
+
+. Click the *Add hosts* button and select the installation media.
+
+.. Select *Minimal image file: Provision with virtual media* to provision the cluster nodes using virtual media. The nodes must have virtual media capability. This option is useful if ISO storage capacity is limited or if the discovery ISO is being served over a constrained network.
+
+.. Select *Full image file: Provision with physical media* to provision with physical media. This is the recommended method.
```

Comment:
I think we changed the defaults, actually making minimal-iso the preferred option